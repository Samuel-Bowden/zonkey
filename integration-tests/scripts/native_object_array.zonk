start {
	let n_class = Input[Input("1234"), Input("4321")];
	n_class.get(0).set_text("first set text");
	n_class.get(1).set_text("second set text");
	println(n_class.get(0).get_text());
	println(n_class.get(1).get_text());
	n_class.push(Input("8765").set_text("third set text"));
	n_class.remove(1);
	println(n_class.get(1).get_text());
}
