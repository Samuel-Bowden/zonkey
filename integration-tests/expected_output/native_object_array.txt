first set text
second set text
third set text
